src/pages/Account/Tabs/ModulesTab.tsx

```diff
@@ -453,21 +459,37 @@ function ModuleSidebar({
       <Typography fontSize={16} fontWeight={500} marginBottom={"24px"}>
         Modules
       </Typography>
-      <Box
-        sx={{
-          maxHeight: "100vh",
-          overflowY: "auto",
-        }}
-      >
-        {moduleNames.map((moduleName, i) => (
-          <ModuleNameOption
-            key={i}
-            handleClick={() => setSelectedModuleIndex(i)}
-            selected={i === selectedModuleIndex}
-            name={moduleName}
-          />
-        ))}
-      </Box>
+      {isWideScreen ? (
+        <Box
+          sx={{
+            maxHeight: "100vh",
+            overflowY: "auto",
+          }}
+        >
+          {moduleNames.map((moduleName, i) => (
+            <ModuleNameOption
+              key={i}
+              handleClick={() => setSelectedModuleIndex(i)}
+              selected={i === selectedModuleIndex}
+              name={moduleName}
+            />
+          ))}
+        </Box>
+      ) : (
+        <FormControl fullWidth>
+          <Select
+            value={selectedModuleIndex}
+            onChange={(e) => setSelectedModuleIndex(Number(e.target.value))}
+            sx={{textTransform: "none"}}
```

Comment:
this doesn't seem to work though ... see "after" screenshot @jillxuu any help?

Reply:
its prob cuz in Select you set the index instead of the value itself, so that `textTransform` wasn't directly applying on the module name